apps/api/lib/view/create.js

```diff
@@ -0,0 +1,31 @@
+import { schema } from '@tpluscode/rdf-ns-builders'
+import { CONSTRUCT } from '@tpluscode/sparql-builder'
+import fromStream from 'rdf-dataset-ext/fromStream.js'
+
+/**
+ * Queries the metadata endpoint to load necessary metadata for a view
+ *
+ * @param req
+ * @param pointer The payload, which is new instance of `</api/View>`
+ */
+export async function importMetadata({ req, pointer }) {
+  const sourceDataset = pointer.out(schema.isBasedOn).term
```

Comment:
From https://schema.org/isBasedOn I understand the predicate is used for adaptations of a source document. for example 'The Rings of Power Series' isBasedOn 'The Silmarillion" 

I see schema:subjectOf or the inverse, schema:about as more adequate for metadata

Reply:
I chose `isBasedOn` because the metadata is actually loaded from the view itself. 

See https://s.zazuko.com/DJyQD

I understand that this is a view itself, but without dimensions and all. Enter view builder. In view builder you will create a `https://view-builder.ld.stadt-zuerich.ch/view/561` and then publish it as a complete view to SSZ database. Again, as  `https://ld.stadt-zuerich.ch/statistics/view/561`

Reply:
I see, it makes sense.

so https://view-builder.ld.stadt-zuerich.ch/view/56 plays as a starting point to build the view.
the uses of the word 'metadata' is very broad :)

Reply:
I discussed this feature with @ktk and it appeared to me that since "our" view and the SSZ view are the same thing. Thus, I thought to replace `schema:isBasedOn` with `schema:sameAs`